components/breadcrumbs/breadcrumb.js

```diff
@@ -0,0 +1,134 @@
+import '../icons/icon.js';
+import '../link/link.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+/**
+ * A component to help users understand where they are within the LMS. would be used as a child of d2l-breadcrumbs component (in slot)
+ * @fires d2l-breadcrumb-connected - Dispatched when the component is initialized to notify parent (so parent can set the compact attribute in event detail)
+ */
+class Breadcrumb extends RtlMixin(LitElement) {
+
+	static get properties() {
+		return {
+			_compact: {
+				type: Boolean,
```

Comment:
Can you also set `attribute: false` so that this won't sprout an attribute and so that setting an attribute won't set this value.

Reply:
Done ✔️